stores/local_storage_store.ts

```diff
@@ -33,85 +33,85 @@ const getPathScopedKey = (path, key) => {
 // Lets open a separate issue to refactor local storage and state interactions.
 // This whole store can be connected to redux
 class LocalStorageStoreClass {
-    getItem(key, state = store.getState()) {
+    getItem(key: string, state = store.getState()) {
         const basePath = getBasePath(state);
 
         return localStorage.getItem(getPathScopedKey(basePath, key));
     }
 
-    setItem(key, value) {
+    setItem(key: string, value: string | null) {
         const state = store.getState();
         const basePath = getBasePath(state);
 
-        localStorage.setItem(getPathScopedKey(basePath, key), value);
+        localStorage.setItem(getPathScopedKey(basePath, key), value === null ? 'null' : value);
```

Comment:
The only runtime change in this file is here. Given that a call like `localStorage.setItem('some string', null); sets the value as `'null'`, this shouldn't be a change in behavior. All other changes in this file are only to add types, and should not influence runtime behavior.

Reply:
Yeah, that's how it would've worked before as well. localStorage would stringify the value we sent anyway, so this is just us replicating the built-in behaviour